### DISCUSSION TITLE
403 error when accessing the end point https://small-long-brook.ropsten.quiknode.pro/

### DISCUSSION THREAD

**Discussion by shukeikoo** [Web3Auth/Web3Auth#1343] [Help]
Since yesterday, our app cannot restore user wallet with web3auto.  It is showing the following error when connecting to the users' google account
Request URL: https://small-long-brook.ropsten.quiknode.pro/e2fd2eb01412e80623787d1c40094465aa67624a
Status Code: 403 
Response: endpoint is disabled at quicknode.com

I tried your demo and see the same error.  Can you please take a look?  Thanks

**Comment by yashovardhan** [comment]
Can you share with us which SDK you're using and which version? Most probably your SDK is out of date which is causing this issue.

Please share some code snippets as well so that we can reproduce the issue.
